Sources/Loco/Analyzer/LocoAnalyser.swift

```diff
@@ -58,7 +51,7 @@ extension LocoAnalyzer {
 			warnings.forEach { error in
                 print(disableColoredOutput ? error : error.coloredDescription)
             }
-			if compileErrors.isEmpty == false {
+			if !compileErrors.isEmpty {
```

Comment:
I saw this a bit late. I prefer isEmpty == false, or true. I think it reads much nicer than !negation.